agent/router/router.go

```diff
@@ -432,6 +458,27 @@ func (r *Router) HasDatacenter(dc string) bool {
 	return ok
 }
 
+func (r *Router) GetManager(areaID types.AreaID, dc string) *Manager {
```

Comment:
I believe this is only used by `GetLANManager`, so we could un-export it for now. Do you think this will need to be exported for ent?

Reply:
Deleted it